terraform/terraform_spec.rb

```diff
@@ -23,6 +23,14 @@
     expect(file("/etc/alpine-release")).to be_file
   end
 
+  it "installs the right version of Alpine" do
+    expect(os_version).to include("Alpine Linux 3.3")
+  end
```

Comment:
I'm not sure I see the value in testing this. Do we care what version of alpine this is? All we care about is that the container has the necessary tools, and that they work.